Sources/EduIDExpansion/Flows/Scanning/ScanViewModelDelegate.swift

```diff
@@ -0,0 +1,10 @@
+import UIKit
+import TiqrCoreObjC
+import AVFoundation
+
+protocol ScanViewModelDelegate: AnyObject {
+    func scanViewModelAddPoints(_for object: AVMetadataMachineReadableCodeObject, viewModel: ScanViewModel)
```

Comment:
missing space: `_ for`

Reply:
I am escaping a reserved word, isn't that correct?

Reply:
Ah, I see. Never seen this style before.